src/py42/services/util.py

```diff
@@ -2,11 +2,12 @@
 
 
 def get_all_pages(func, key, *args, **kwargs):
-    item_count = settings.items_per_page
-    page_num = 0
     if kwargs.get("page_size") is None:
         kwargs[u"page_size"] = settings.items_per_page
-    while item_count >= settings.items_per_page:
+
+    item_count = page_size = kwargs[u"page_size"]
```

Comment:
Can we add a test for this?

Reply:
planning on it, just wanted to make this available for physical testin

Reply:
@unparalleled-js added!